.github/workflows/releaseChart.yaml

```diff
@@ -1,8 +1,15 @@
 name: Release newrelic prometheus configurator chart
+# This workflow manually triggers the automated process of release based on the changelog.
+# Is recommended to run `make release-changelog` before to see an example of what is going to be generated.
+
+# IMPORTANT No matter the branch selected when triggered, the workflow will always execute on 'main' branch.
+
```

Comment:
we are adding a check to be sure we are running over main, @gsanchezgavier is taking care of it

Reply:
leaving this open to remember to add it before merging